src/components/static-map.js

```diff
@@ -141,6 +143,11 @@ export default class StaticMap extends PureComponent {
       canvas.style.outline = 'none';
     }
 
+    // Attach optional onLoad function
+    if (this.props.onLoad) {
```

Comment:
We could add a `noop` function in `defaultProps`, then we can call the callback here without an if statement.